plugins/mapper-annotated-text/src/test/java/org/elasticsearch/search/highlight/AnnotatedTextHighlighterTests.java

```diff
@@ -65,10 +65,10 @@ private void assertHighlightOneDoc(String fieldName, String []markedUpInputs,
         
         // Annotated fields wrap the usual analyzer with one that injects extra tokens
         Analyzer wrapperAnalyzer = new AnnotationAnalyzerWrapper(new StandardAnalyzer());
-        AnnotatedHighlighterAnalyzer hiliteAnalyzer = new AnnotatedHighlighterAnalyzer(wrapperAnalyzer);
-        hiliteAnalyzer.init(markedUpInputs);
-        PassageFormatter passageFormatter = new AnnotatedPassageFormatter(hiliteAnalyzer,new DefaultEncoder());
-        String []plainTextForHighlighter = hiliteAnalyzer.getPlainTextValuesForHighlighter();
+        AnnotatedHighlighterAnalyzer highlightAnalyzer = new AnnotatedHighlighterAnalyzer(wrapperAnalyzer);
```

Comment:
I know some people are attached to the alternate spelling, but this repository mostly uses the long format.